.github/workflows/auto_approve_pull_requests.yml

```
@@ -1,16 +1,84 @@
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Verify status checks
+        id: check-status
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}  # Token for rocketmq-rust-bot account
+          script: |
+            const { owner, repo } = context.repo;
+            const pr = context.payload.pull_request || context.payload.check_suite.pull_requests[0];
+            const sha = pr.head.sha;
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)